content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -593,6 +603,15 @@ This code set contains levels of severity (mild, moderate, severe) used by multi
       "display": "Moderate",
     }
 
+##### Code Set 14002 Bill Code Schedule
+
+This code set contains group of bill codes that drives billing behavior.
```

Comment:
Let's use something like "This code set contains bill code schedules that drive billing behavior."

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/629/commits/8a612215cd46509787f4a54f29c6120975f44c27#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R608

Reply:
+1